settings/DevHome.Settings/Strings/en-us/Resources.resw

```diff
@@ -165,7 +165,7 @@
     <value>Install a Dev Home extension that uses your developer account, then sign in to add the account.</value>
   </data>
   <data name="Settings_Accounts_NoProvidersContentDialog_PrimaryButtonText" xml:space="preserve">
-    <value>OK</value>
+    <value>Find Extensions</value>

```

Comment:
I would expect they'd want a lowercase e in extensions

Reply:
It was upper case in the specs, but I will ask them to confirm.